toc/working-groups/app-runtime-interfaces.md

```diff
@@ -510,6 +510,28 @@ areas:
   - cloudfoundry/notifications
   - cloudfoundry/app-runtime-interfaces-infrastructure
 
+- name: Stratos Console for Cloud Foundry
+   approvers:
```

Comment:
PR validation failed. Looks to me as if yaml indentation is not correct (approvers etc should be on same level as name)

Reply:
Fixed indentation and tested with spruce.